### PR TITLE
tests for glmnet objects from glmnetUtils package; closes #473

### LINKED ISSUE #473: Add tests for glmnetUtils objects

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: tests/testthat/test-glmnetUtils.R [hunk 1]
```diff
@@ -0,0 +1,63 @@
+context("glmnetUtils")
+
+skip_if_not_installed("modeltests")
+library(modeltests)
+
+skip_if_not_installed("glmnetUtils")
+library(glmnet)
```
Comment (alexpghayes) [inline tests/testthat/test-glmnetUtils.R]: These tests don't use the `glmnetUtils` package or a formula interface?

Comment (IndrajeetPatil) [reply tests/testthat/test-glmnetUtils.R]: Done. None of the relevant tests fail.

File: tests/testthat/test-glmnetUtils.R [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+set.seed(27)
+
+x <- matrix(rnorm(100 * 20), 100, 20)
+y <- rnorm(100)
+g <- sample(1:4, 100, replace = TRUE)
+
+fit <- glmnet(x, y)
+fit2 <- glmnet(x, g, family = "multinomial")
+
+cv_fit <- cv.glmnet(x, y)
```
Comment (alexpghayes) [inline tests/testthat/test-glmnetUtils.R]: Can you add tests that explicitly use the formula interface rather than the matrix interface?

**Comment by IndrajeetPatil** [comment]
@alexpghayes Done!

**Comment by alexpghayes** [comment]
Thanks!

**Pull request merged**